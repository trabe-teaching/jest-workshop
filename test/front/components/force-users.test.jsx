import React from "react";
import { shallow } from "enzyme";
import ForceUsers from "components/force-users";

describe("front/components/force-users", () => {
  const users = [
    {
      id: 1,
      name: "Luke Skywalker",
    },
    {
      id: 2,
      name: "Yoda",
    },
  ];

  it("renders the force users", () => {
    const wrapper = shallow(<ForceUsers users={users} onX="onX" onY="onY" />);
    expect(wrapper).toMatchSnapshot();
  });
});
