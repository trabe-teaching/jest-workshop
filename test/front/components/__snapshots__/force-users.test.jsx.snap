// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`front/components/force-users renders the force users 1`] = `
<React.Fragment>
  <ForceUser
    key="1"
    onX="onX"
    onY="onY"
    user={
      Object {
        "id": 1,
        "name": "Luke Skywalker",
      }
    }
  />
  <ForceUser
    key="2"
    onX="onX"
    onY="onY"
    user={
      Object {
        "id": 2,
        "name": "Yoda",
      }
    }
  />
</React.Fragment>
`;
